src/common/helpers/formatPriceRequestVoteData.ts

```diff
@@ -151,6 +188,7 @@ export default function formatPriceRequestVoteData(
       uniqueClaimers: uniqueClaimers.toString(),
       uniqueClaimersPctOfReveals: uniqueClaimersPctOfReveals.toString(),
       time: Number(rr.time),
+      voterRewards,
```

Comment:
from my testing these rewards get shown when claimed, so this isnt actually needed. is it ok to always do this calculation even if its not needed since rewards claimed is available?

Reply:
So you want to conditionally do this check?